src/pages/Menu.tsx

```diff
@@ -1,5 +1,8 @@
 const Menu = () => {
-	return <h2>Menu</h2>
+	return (
```

Comment:
this change has nothing to do with your PR. why is it here?

Reply:
Maybe some `spase` missed